packages/scss/src/commons/config.scss

```diff
@@ -347,7 +347,7 @@ $sizes: (
 	XL: (
 		fontSize: 1.375rem,
 		lineHeight: 1.75rem,
-		verticalPadding: 0.5rem,
+		verticalPadding: 0.375rem,
```

Comment:
why do we need to still add padding, instead of raise the lineHeight ?

Reply:
For text that passes to the line.